content/v2.0/security/enable-tls.md

```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
-With this certificate option, every InfluxDB instance requires a unique single domain certificate.
+    Single domain certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    These certificates are signed and issued by a trusted, third-party Certificate Authority (CA).
+    With this certificate option, every InfluxDB instance requires a unique single domain certificate.
 
-### Wildcard certificates signed by a Certificate Authority
+* **Wildcard certificates signed by a Certificate Authority**
 
-Wildcard certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
-Wildcard certificates can be used across multiple InfluxDB instances on different servers.
+    Wildcard certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    Wildcard certificates can be used across multiple InfluxDB instances on different servers.
 
-### Self-signed certificates
+* **Self-signed certificates**
 
-Self-signed certificates are _not_ signed by a trusted, third-party CA.
-Unlike CA-signed certificates, self-signed certificates only provide cryptographic security to HTTPS requests.
-They do not allow clients to verify the identity of the InfluxDB server.
-With this certificate option, every InfluxDB instance requires a unique self-signed certificate.
-You can generate a self-signed certificate on your own machine.
+    Self-signed certificates are _not_ signed by a trusted, third-party CA.
+    Unlike CA-signed certificates, self-signed certificates only provide cryptographic security to HTTPS requests.
+    They do not allow clients to verify the identity of the InfluxDB server.
+    With this certificate option, every InfluxDB instance requires a unique self-signed certificate.
+    You can generate a self-signed certificate on your own machine.
 
 <!-- InfluxDB supports certificates composed of a private key file (`.key`) and a signed certificate file (`.crt`) file pair, -->
 <!-- as well as certificates that combine the private key file and the signed certificate file into a single bundled file (`.pem`). -->
 
-## Enable HTTPS with a CA-signed certificate
+## Setup InfluxDB to use TLS over HTTPS
+
+1. **Download or generate certificate files**
 
-1. **Install the certificate**
+    If using a certificate provided by a CA, follow their instructions to download the certificate files.
 
-    Place the private key file (`.key`) and the signed certificate file (`.crt`) in the `/etc/ssl/` directory.
+    If using a self-signed certificate, use the `openssl` utility to create a certificate.
+    The following command generates a private key file (`.key`) and a self-signed
+    certificate file (`.crt`) which remain valid for the specified `NUMBER_OF_DAYS`.
+    It outputs those files to `/etc/ssl/` and gives them the required permissions.
```

Comment:
If using a self-signed...., run the following to create a certificate:
```sh
.....
```
what info does command prompt for...valid number of days? directory to save files? 
`openssl` generates a private key file (`.key`) and a self-signed certificate file (`.crt`) with required permissions and saves them to `/etc/ssl/`. Files remain valid for the specified `NUMBER_OF_DAYS`.

Reply:
@pierwill what do you think about this one?

Reply:
Missed it on the first time around, looking at it now—